ios/brave-ios/scripts/scheme_preaction.py

```diff
@@ -43,8 +43,8 @@ def main():
     target_environment = 'simulator' if (options.platform_name
                                          == 'iphonesimulator') else None
 
-    UpdateSymlink(config, target_arch, target_environment)
     BuildCore(config, target_arch, target_environment)
+    UpdateSymlink(config, target_arch, target_environment)
     GenerateXCFrameworks(config, target_arch, target_environment)
     CleanupChromiumAssets(output_dir)
     FixMaterialComponentsVersionString(output_dir)
```

Comment:
all these commands should be converted into GN actions.

this script should only call `BuildCore` and `UpdateSymlink`.

Reply:
Opened follow up issue here for this https://github.com/brave/brave-browser/issues/35865